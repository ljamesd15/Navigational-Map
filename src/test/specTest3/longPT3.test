# This is in attempt to find paths relatively long pt3

# Load graph
LoadGraph marvel marvel.tsv

# Find the path
FindPath marvel FLARE/ GENUARDI,_MR.