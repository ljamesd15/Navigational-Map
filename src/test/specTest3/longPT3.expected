# This is in attempt to find paths relatively long pt3

# Load graph
loaded graph marvel

# Find the path
path from FLARE/ to GENUARDI, MR.:
FLARE/ to SPIDER-MAN CLONE/BEN with weight 1.000
SPIDER-MAN CLONE/BEN to SPIDER-MAN/PETER PAR with weight 0.010
SPIDER-MAN/PETER PAR to CAPTAIN AMERICA with weight 0.007
CAPTAIN AMERICA to GENUARDI, MR. with weight 0.500
total cost: 1.517